spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/writer/WriteConf.scala

```diff
@@ -54,17 +54,22 @@ object WriteConf {
   val DefaultBatchBufferSize = 1000
   val DefaultBatchLevel = BatchLevel.Partition
   val DefaultThroughputMiBPS = Int.MaxValue
+  import CassandraConnectorConf.convert
 
-  def fromSparkConf(conf: SparkConf): WriteConf = {
+  def fromSparkConf(conf: SparkConf): WriteConf = getWriteConf(conf, None)
+
+  def fromSparkConf(conf: SparkConf, cluster: Option[String]): WriteConf = getWriteConf(conf, cluster)
```

Comment:
I would do the same with `WriteConf` what I would do for `ReadConf`. The only properties which are cluster-related are in `CassandraConnector`.